pkgs/sops-import-keys-hook/default.nix

```diff
@@ -1,8 +1,9 @@
 { stdenv, makeSetupHook, gnupg, sops, nix }:
 
 (makeSetupHook {
+  name = "sops-import-keys-hook";
   substitutions = {
     gpg = "${gnupg}/bin/gpg";
   };
-  deps = [ sops gnupg ];
+  propagatedBuildInputs = [ sops gnupg ];
```

Comment:
Looks like this breaks nixos 22.11?

Reply:
Yeah, not sure what's the best way to go about this deprecation, as `propagatedBuildInputs` does not exist in earlier nixpkgs.

Reply:
@Tomaszal do you want to implement this change?